app/scripts/modules/core/src/index.ts

```diff
@@ -48,6 +48,8 @@ export * from './managed';
 
 export * from './manifest';
 
+export * from './mocks';
```

Comment:
I'm not sure we want the mocks exported from (and bundled into) `@spinnaker/core`.  @caseyhebebrand what do you think?

Reply:
This was one thing I wanted thoughts on. We could either:
1. Keep mocks tied closely with the `domain` folder of a Spinnaker package, therefore living in various packages. 
2. Create one `mocks` subdirectory that references types from all areas of Spinnaker. 

I am more in favor of #2, so tests do not have to be bundled to `spinnaker/core` if they don't have to be. I just wasn't sure if people would be in favor of relying on another package.

Reply:
I don't know what my opinion is for 1 vs 2

my question is more about the re-export here.  By exporting the mocks at the top level `index.ts`, it guarantees that the mock code is included in the production core bundle.  However, we only want to use mocks in tests.  

I think this export should be removed from `index.ts`.  Tests will still need access, but I think they can `import { fooMock } from '@spinnaker/core/mocks'`